packages/serverpod/lib/src/server/session.dart

```diff
@@ -110,11 +110,7 @@ abstract class Session {
 
   bool _closed = false;
 
-  /// True if logging is enabled for this session. Normally, logging should be
-  /// enabled but it will be disabled for internal sessions used by Serverpod.
-  final bool enableLogging;
```

Comment:
Suggestion: This cannot be removed as this is part of the public interface.

Reply:
Ah of course 👍 Thanks!

Reply:
Reverted here: [cec5fad](https://github.com/serverpod/serverpod/pull/2676/commits/cec5fad3d1dececc42a205ca6662da6c4342f464)